_events/2023/2023-03-education-training-tutorial.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Setting up a Scientific Software Project using GitLab -- From Zero to Hero
+subtitle: US-RSE Education & Training Seminar Series
+expires: 2023-03-22
```

Comment:
I never remember, does the expire time have to be the day after the event or is the same day ok?

Reply:
I used the model from previous ones. I am not 100% sure. But, this seems to be the way previous events worked.